### PR TITLE
Remove system-config

### PR DESCRIPTION
**Pull request by jpdagostino**

### CODE REVIEW

File: client/src/game/localStorage.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export function loadGameModes(): NonNullable<unknown> | null {
         try {
             return JSON.parse(data);
         } catch {
+            localStorage.clear();
```
Comment (Jack-Papel) [inline client/src/game/localStorage.tsx:64]: This isn't the only thing saved in local storage. Just clear the specific item

File: LICENSE
Comment (Jack-Papel) [inline LICENSE:1]: Seems unrelated. Accidentally included from your other branch? This should be removed from this PR

**Review by Jack-Papel — CHANGES_REQUESTED** [review]

File: LICENSE
Comment (jpdagostino) [reply LICENSE:1]: Wait why is it automatically pulling other commits
